### PR TITLE
NC-127 General LDAP-Self Service aka SC-795

### PR DESCRIPTION
**Pull request by malcm**
minor changes for better usability should be added. Otherwise self service is now possible for LDAP systems

### CODE REVIEW

File: static/scripts/administration/systems/ldap-edit.js [hunk 1]
```diff
@@ -17,15 +16,183 @@ function toggleFieldClass(clicked, fieldClass) {
...
+
+	$.ajax({
+		type: 'POST',
+		url: window.location.href,
+		data: $('#ldap-form').serialize(),
+		dataType: 'json',
+		timeout: 10000, // sets timeout to 10 seconds
+		error: function(){
+			// will fire when timeout is reached
+			$.showNotification("LDAP Zeitüberschreitung - Mögliche Gründe sind: falsche Server-Daten, Nicht offiziell signierte SSL Zertifikate (ldaps), flasche Nutzerdaten für den search-Nutzer, inkorrekter root-Pfad, inkorrekter Nutzer-Pfad, inkorrekter Klassen-Pfad oder inkorrekte Rollen-LDAP-Pfade. Sollten Sie das Problem nicht lösen können, kontaktieren Sie den Support.", "danger");
```
Comment (Parnswir) [inline static/scripts/administration/systems/ldap-edit.js]: typo 🍾

File: views/administration/forms/form-ldap-edit.hbs [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
 <div class="form-group">
     <label>search-Nutzer Passwort</label>
-    <input class="form-control linked" name="searchuserpassword" type="password" value=""/>
+    <input class="form-control linked" name="searchuserpassword" type="password" value="{{system.ldapConfig.searchUserPassword}}"/>
 </div>
 
 <div class="form-group">
-    <label>Nutzer-Pfad <i class="fa fa-question-circle"></i></label>
-    <input class="form-control linked" name="userpath" type="text" value=""/>
+    <label>Nutzer-Pfad <i class="fa fa-question-circle" data-toggle="tooltip" title="Relativer Pfad vom rootPath zu dem LDAP-Knoten von dem alle Nutzer erreichbar sind."></i></label>
```
Comment (Parnswir) [inline views/administration/forms/form-ldap-edit.hbs]: Hier heißt es "rootPath", aber weiter oben heißt es "root-Pfad"

File: static/scripts/administration/systems/ldap-edit.js (hunk 1 above)
Comment (malcm) [reply static/scripts/administration/systems/ldap-edit.js]: Can you fix it?

Comment (Parnswir) [reply static/scripts/administration/systems/ldap-edit.js]: done

**Pull request merged**